src/app/resources/resources-add.component.ts

```diff
@@ -46,7 +46,7 @@ export class ResourcesAddComponent implements OnInit {
   file: any;
   resourceForm: FormGroup;
   readonly dbName = 'resources'; // make database name a constant
-
+  ischecked = false;
```

Comment:
isChecked is part of formGroup, so this here is unnecessary and should be removed